test/win/valhalla.json

```diff
@@ -194,8 +194,8 @@
       "min_resample": 10.0
     },
     "trace": {
-      "max_best_paths": 4,
-      "max_best_paths_shape": 100,
+      "max_alternates": 4,
```

Comment:
this file hasn't been kept up to date for a long while it seems... there are probably more entries missing but these changes were necessary for the smoke test to work